modules/database/variables.tf

```diff
@@ -26,3 +26,23 @@ variable "deletion_protection" {
   type        = bool
   default     = true
 }
+
+variable "apply_pitr_backup_rds" {
```

Comment:
This can be removed, just allow users to specify the variables in the same way they provide them [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#restore-to-point-in-time).

e.g. they just specify
- `rds_restore_to_point_in_time_restore_time`
- `rds_restore_to_point_in_time_source_db_instance_identifier`